pkg/scheduler/schedulercache/cache.go

```diff
@@ -313,7 +313,7 @@ func (cache *schedulerCache) RemovePod(pod *v1.Pod) error {
 		}
 		delete(cache.podStates, key)
 	default:
-		return fmt.Errorf("pod state wasn't added but get removed. Pod key: %v", key)
+		return fmt.Errorf("pod %v wasn't added so cannot be removed", key)
```

Comment:
Please mention cache here:
"pod %v is not found in the scheduler cache, so cannot be removed from it"